### PR TITLE
Port module sale_comment_template to V10

### PR DESCRIPTION
**Pull request by clb-openfire**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,12 +9,17 @@ You'll find modules that implement various report related to sales in general.
 
 [//]: # (addons)
 
+Available addons
+----------------
+addon | version | summary
+--- | --- | ---
+[sale_comment_template](sale_comment_template/) | 10.0.1.0.0 | Comments texts templates on Sale documents
```
Comment (pedrobaeza) [inline README.md]: Don't change this. It's done automatically.

File: sale_comment_template/views/base_comment_template_view.xml [hunk 2]
```diff
@@ -1,10 +1,10 @@
 <?xml version="1.0" encoding="utf-8"?>
-<openerp>
+<odoo>
   <data noupdate="0">
```
Comment (SimoRubi) [inline sale_comment_template/views/base_comment_template_view.xml:3]: Remove data tag with noupdate="0" as stated in [Migration-to-10 guide](https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-10.0)

File: sale_comment_template/views/sale_order_view.xml [hunk 3]
```diff
@@ -1,32 +1,30 @@
...
   <data>
     <record model="ir.ui.view" id="sale_order_form_add_comment">
       <field name="name">comment_template.sale.order.form</field>
       <field name="model">sale.order</field>
       <field name="inherit_id" ref="sale.view_order_form"/>
       <field name="arch" type="xml">
-        <xpath expr="//page[@string='Other Information']" position="after">
+        <xpath expr="//notebook" position="inside">
           <page string="Comments" name="comments">
             <label string="The comments will be displayed on the printed document. You can load a predefined template, write your own text or load a template and then modify it only for this document." colspan="2"/>
```
Comment (SimoRubi) [inline sale_comment_template/views/sale_order_view.xml:10]: Labels are different in 10.0 from how they were in 8.0 (now they are bold); consider to change it to a p and maybe to wrap it in a group for appropriate padding from notebook tabs.

**Review by SimoRubi — CHANGES_REQUESTED** [review]
Moreover, this isn't working in my local environment, with the following steps:
1. created sale order with comments
2. Action -> Invoice order
The created invoice has no comments. Please also add tests to prevent regressions

**Comment by pedrobaeza** [comment]
Superseeded by #44

**Pull request closed without merging**